doc/man3/flux_shell_service_register.adoc

```diff
@@ -0,0 +1,47 @@
+flux_shell_service_register(3)
+==============================
+:doctype: manpage
+
+NAME
+----
+flux_shell_service_register - Register a service handler for `method` in the shell.
+
+SYNOPSIS
+--------
+
+ #include <flux/shell.h>
+
+ int flux_shell_service_register (flux_shell_t *shell, 
+                                  const char *method, 
+                                  flux_msg_handler_f cb, 
+                                  void *arg);
+
+DESCRIPTION
+-----------
+
+Wrapper for `shell_svc_register()`.  
```

Comment:
`shell_svc_register()` is a shell-internal function. A short description of this function could be:

```suggestion
The job shell registers a unique service name with the flux broker on startup, and posts the
topic string for this service in the context of the `shell.init` event. 
`flux_shell_service_register()` allows registration of a request handler `cb` for subtopic
`method`  on this service endpoint, allowing other job shells and/or flux commands to
interact with arbitrary services within a job.
```

Reply:
Done!